### ISSUE TITLE
Issues training the ytvis_2019 model

### ISSUE THREAD

**Issue by ntuLC** [facebookresearch/Mask2Former#24]
Hi Bowen, 

Thanks for your excellent work and code! I am retraining the video instance segmentation model on the Youtube VIS 2019 dataset. I managed to train the model, but the quantitative result on the CodaLab turns to be very low (of only about 40). 

The command I used for training is: 

```
python3 train_net_video.py \
    --config-file configs/youtubevis_2019/swin/video_maskformer2_swin_base_IN21k_384_bs16_8ep.yaml \
    --num-gpus 8 \
    MODEL.WEIGHTS swin_base_patch4_window12_384_22k.pkl
```

The backbone weight is got from:

```
wget https://github.com/SwinTransformer/storage/releases/download/v1.0.0/swin_base_patch4_window12_384_22k.pth
python tools/convert-pretrained-swin-model-to-d2.py swin_base_patch4_window12_384_22k.pth swin_base_patch4_window12_384_22k.pkl
```

Then I submitted the `output/inference/results.json` after the training to CodaLab, but only got 40 accuracy. I also tried to rerun the evaluation using `output/model_final.pth`, and the results are almost the same.

The config files are untouched. I am able to reproduce the correct result using the pretrain model, so I assume that my environment and dataset setups are ok. I also checked the tensorboard output and the loss curve looks good. Could you help to check if there is anything wrong with my training process? Thanks!

**Comment by bowenc0221** [comment]
Please use Mask2Former COCO instance segmentation pre-trained checkpoints.

**Comment by ntuLC** [comment]
Got it. Thanks for your reply!

**Comment by superaha** [comment]
Hi @bowenc0221 ,

I have used the pretrained coco model to initialize the video model on ytvis_2019. However, the mAP is about 50 using swin-base, still much slower than the results in the paper. 

However, when loading the pretrained model, I do see the following warning. Do you have any suggestions where the issues may come from?

<img width="1461" alt="image" src="https://user-images.githubusercontent.com/76199834/151093468-75d2fe72-3323-4e38-b6b2-4de604904f3b.png">

Thanks!
